src/fragments/lib-v1/storage/flutter/dev-preview-api-callout.mdx

```diff
@@ -1,5 +1,5 @@
 <Callout warning>
 
-This API is only available in developer preview. See the documentation for [project setup](/lib-v1/project-setup/create-application) for more information about getting started.
+This API is available in the v1 version. Use the version switcher to see v1 documentation.
```

Comment:
Hui has deleted this file in his PR. I don't think it's needed anymore since the pages for these APIs are being removed.

Reply:
Yes this callout is no longer applicable in v0 doc, so I removed it.